PhpAmqpLib/Wire/GenericContent.php

```diff
@@ -126,7 +122,7 @@ public function load_properties($r)
         while (true) {
             $flag_bits = $r->read_short();
             $flags[] = $flag_bits;
-            if (($flag_bits & 1) == 0) {
+            if (($flag_bits & 1) === 0) {
```

Comment:
`===` enforces type-checking in the comparison.  So `"0" === 0` returns false.  I think it's always good to do when able.  Makes your code more stable.   PHP's loosly typed-ness is nice in that it's more forgiving and a great feature, but can also give you unforseen results.  So I like to err on the side of strict typing by default with the option of loose type checking as a feature if I need it.